src/Moq/ExpressionExtensions.cs

```diff
@@ -169,4 +169,25 @@ internal static (Expression Object, MethodInfo Method, IReadOnlyList<Expression>
 				expression.ToStringFixed()));
 		}
 	}
+
+	internal readonly struct CallInfo
```

Comment:
We *could* declare a few generic tuple types instead of this and other concrete `readonly struct`s. For now, that degree of reusability isn't needed, and we spare the JITter some additional work by avoiding generics. (And, perhaps more importantly, declaring concrete, non-generic types gives us a means of giving the constituent parts names more meaningful than just `Item1`, `Item2`, etc.)